src/identity/Identity.ts

```diff
@@ -100,20 +202,76 @@ export default class Identity extends PublicIdentity {
 
   private readonly signKeyringPair: KeyringPair
   private readonly boxKeyPair: BoxKeyPair
+  /**
+   * Creates a new instance of public identity.
+   * @returns `address`
+   * @returns `boxPublicKeyAsHex`
+   */
 
   public getPublicIdentity(): PublicIdentity {
     const { address, boxPublicKeyAsHex } = this
     return { address, boxPublicKeyAsHex }
   }
 
+  /**
+   * Adds the signing for a message with an Identity using signing key ring pairs.
+   * @param cryptoInput
+   * @returns `Crypto`
```

Comment:
Fix: this returns the output of `Crypto.sign(...)`

Reply:
Changed this out and updated the examples